pkgs/development/libraries/mesa/default.nix

```diff
@@ -57,7 +57,7 @@ stdenv.mkDerivation {
   patches = [
     ./missing-includes.patch # dev_t needs sys/stat.h, time_t needs time.h, etc.-- fixes build w/musl
     ./opencl-install-dir.patch
-    ./disk_cache-include-dri-driver-path-in-cache-key.patch
```

Comment:
We should check carefully if this patch is indeed not required anymore. From a quick look at the source-code it looks ok, but I haven't checked all drivers yet. Some still refer to the parameter as `*timestamp` (inconsistently even `src/util/disk_cache.h`) while using the build-id.

I guess I'm mainly concerned since https://github.com/NixOS/nixpkgs/issues/91145 apparently didn't affect all drivers.

Reply:
I did a somewhat thorough check.  All the dri and vulkan drivers have build-id.

`lib/dri/radeonsi_drv_video.so` does not have build-id, and does contain `si_disk_cache_create`.  I'm not sure if that's a problem.  None of the `_drv_video.so` libs or vdpau libs have build-id.

The problem with the cache key patch is that it only affects the actual disk cache keys, not other uses of `cache_uuid`, such as the one in radv.

I'd be happy just to merge a23c6888d399b291ba691e43028e1b295ad10efc while this gets investigated more.

Reply:
This seems to be the only driver that still uses timestamps directly:

https://gitlab.freedesktop.org/mesa/mesa/-/blob/master/src/freedreno/vulkan/tu_device.c#L61